src/Font.ts

```diff
@@ -0,0 +1,150 @@
+import {
+	compressSpaces
+} from './util';
+
+function wrapFontFamily(fontFamily: string) {
+
+	const trimmed = fontFamily.trim();
+
+	return /^('|")/.test(trimmed)
+		? trimmed
+		: `"${trimmed}"`;
+}
+
+function prepareFontFamily(fontFamily: string) {
+	return typeof process === 'undefined'
```

Comment:
should probably just take care of this in a build step instead of checking at runtime if it's node or not

Reply:
Generate two separate bundles for node and browsers because of this one line?

Reply:
we currently generate 2 bundles - especially when we want to target different ecma support - i also assume there are other areas in which we're sniffing too or is that not correct? most of the code does behave uniformly and if there aren't too many of these places i can be convinced to switch to runtime checks. maybe we can at least move all of them behind a `NODE_ENV` flag where `NODE_ENV: bool = process !== 'undefined'` or something